.gitignore

```diff
@@ -3,4 +3,6 @@ node_modules
 logs
 dist
 .env
-application.log
\ No newline at end of file
+application.log
+package-lock.json
```

Comment:
@nairicode 

https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

Reply:
ok i will remove from .gitignore, just i have seen in other packages where package-lock.json in .gitignore
for example https://github.com/hapijs/hapi/blob/master/.gitignore#L17